report/renderer/topbar-features.js

```diff
@@ -273,20 +254,39 @@ export class TopbarFeatures {
     }
   }
 
-  _setupStickyHeaderElements() {
+  _setupStickyHeader() {
+    // Cache these elements to avoid qSA on each onscroll.
     this.topbarEl = this._dom.find('div.lh-topbar', this._document);
-    this.scoreScaleEl = this._dom.find('div.lh-scorescale', this._document);
-    this.stickyHeaderEl = this._dom.find('div.lh-sticky-header', this._document);
+    this.categoriesEl = this._dom.find('div.lh-categories', this._document);
+
+    // Only present in the DOM if it'll be used (>=2 categories)
+    try {
+      this.stickyHeaderEl = this._dom.find('div.lh-sticky-header', this._document);
+    } catch (e) {}
+
+    if (!this.stickyHeaderEl) return;
 
     // Highlighter will be absolutely positioned at first gauge, then transformed on scroll.
     this.highlightEl = this._dom.createChildOf(this.stickyHeaderEl, 'div', 'lh-highlighter');
+
+    // Defer behind rAF to avoid forcing layout
+    requestAnimationFrame(() => requestAnimationFrame(() => {
```

Comment:
Why double `requestAnimationFrame`?

Reply:
`getScrollParent` forces layout. and since it currently happens during report creation, we have   TWO 50ms blocks of style recalc (one forced and one normal async).

since this code can run a little later without consequence.... i figured deferring it was nice :)
doubleRaf specifically because w/ singleRaf it can still happen in the same frame (and thus still have double the rendering costs).

this would be easier to communicate had i kept the traces when i was doing this.. but alas they gone. :p